build.sh

```diff
@@ -29,6 +29,8 @@ cd -
 
 cd contracts/assemblyscript/erc20
 rm -rf build
-yarn && yarn build
+yarn
+yarn add assemblyscript
```

Comment:
Could you add it to all three assemblyscript contracts, the bug can occur in all three cases. Thanks!

Reply:
(I can actually also do that in a second PR later)